### PR TITLE
Add Gitlab to newcomer's guide

### LINKED ISSUE #3901: Mention GitLab coala in newcomers guide
difficulty/newcomer

Opened via [gitter](https://gitter.im/coala/coala/?at=58c4cdd8e961e53c7fd39e75) by @jayvdb

### PR DESCRIPTION
**Pull request by bhavishyagopesh**
Added Gitlab to newcomer's guide.

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

Fixes #3901

### CODE REVIEW

**Comment by bhavishyagopesh** [comment]
@coala-bot can anyone merge this?

**Comment by NiklasMM** [comment]
Something is not right here. Someone elses commit is in your PR. Make sure your branch is correctly rebased and only your commit is included: http://www.coala.io/rebase

Also are you fixing any issue? If so, please reference the issue both in the PR and in the commit message.

**Comment by bhavishyagopesh** [comment]
2b8f6458d32370585ed037a11d46194b8203b6a3 Adds Gitlab mention in newcomer's guide.

**Comment by Techievena** [comment]
Also please follow commit guidelines coala.io/commit

**Comment by bhavishyagopesh** [comment]
Sorry, I'm running short of time for patch requirement so I'm currently working on #3882 ,Can I see it in a while.

**Comment by NiklasMM** [comment]
I would _really_ encourage you to finish this PR, before you move on to the next issue. Understanding and going through with the review process is crucial if you want to contribute to coala. A small documentation issue like this is perfect to learn the basics (commit messages, rebasing, etc).

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -119,6 +119,8 @@ If there's anything unclear, or you are encountering problems, feel free
 to contact us on `gitter <https://coala.io/chat>`_,
 and we will help you!
 
+Optional. coala also has some of its repositories on  `Gitlab <https://gitlab.com/coala>`_.
```
Comment (userzimmermann) [inline docs/Developers/Newcomers_Guide.rst]: GitLab is more or less the future than just another option...

**Comment by userzimmermann** [comment]
@bhavishyagopesh Hey :) And C'mon ;) Just make those few requested changes and your contribution is fine and helpful!

**Comment by jayvdb** [comment]
Note that the patch author doesn't have the same username on GitLab: https://gitlab.com/bhavishyagopesh (and username search on GitLab is a bit borked)

I have emailed the patch author just now.  If there isnt some traction soon, the PR should be closed and the issue re-assigned.

**Comment by jayvdb** [comment]
Solved by someone else.

**Pull request closed without merging**